markdown_lint/linter.py

```
@@ -0,0 +1,317 @@
+
+            if report.fixed_content is not None:
+                try:
+                    with open(file_path, "w", encoding="utf-8") as f:
+                        f.writelines(report.fixed_content)
+                    fixed_count += 1
+                    print(
+                        f"Fixed {len([i for i in report.issues if i.fixable])} "
+                        f"issues in {file_path}"
+                    )
+                except Exception as e:
+                    print(f"Error fixing {file_path}: {e}", file=sys.stderr)
```

Ruff (0.12.2)

Undefined name `sys`

(F821)